### PR TITLE
Tools and services/preferred services update

### PR DESCRIPTION
**Pull request by hcarreras**
The first round of preferred services.
The three-color system will allow us to decide what to do in each case.

Next steps
More policies will follow in regards to:
- How to categorize services
- What to do when using red services

### CODE REVIEW

**Comment by jeppester** [comment]
Looks good to me.
It's way better than what we had before and we can always expand.

I think we should get the dotnet and WP folks to chime in with their knowledge about azure, kinsta etc.

**Comment by KlausEnevoldsen-Abtion** [comment]
Looks good. Do we add tools/addons as we go or should we schedule some time to update the list with what we use now?

File: tools_and_services/preferred_services.md [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
+If there is a service in this category, but the team wants to use a different one, the team must first check with the Principal Engineer or CTO.
+- ![](https://via.placeholder.com/10/ffff00/?text=+) Yellow list: We've used the services before, and we are satisfied. The services in this list should be the first option to consider on a team, although the team can decide to use a different tool if there is a good reason behind it.
+- ![](https://via.placeholder.com/10/f03c15/?text=+) Red list: The services in this list have been used or analyzed, and they do not live up to Abtion standard. A team must request permission from the Principal Engineer or CTO before using this tool on a production system.
+
+
+### Sending and receiving SMS
+![](https://via.placeholder.com/10/00c900/?text=+) [InMobile](https://www.inmobile.com/)
+
+### Error management
+![](https://via.placeholder.com/10/ffff00/?text=+) [Sentry](sentry.io) In Heroku: use the addon.
```
Comment (runephilosof-abtion) [inline tools_and_services/preferred_services.md:17]: Rollbar is automatically added and configured by Rails Muffi

File: tools_and_services/preferred_services.md [hunk 2]
```diff
@@ -1,15 +1,36 @@
...
+### Mailing
+![](https://via.placeholder.com/10/ffff00/?text=+) [SendGrid](https://sendgrid.com/), in Heroku: use the addon (Automatically added by [Muffi](https://github.com/abtion/muffi)).
 Remember to set up DKIM (under sender authentication -> domain authentication).
 
+![](https://via.placeholder.com/10/f03c15/?text=+) [MailGun](mailgun.com) Countless issues when using the addon on Heroku. Accounts are banned on regular basis.
```
Comment (runephilosof-abtion) [inline tools_and_services/preferred_services.md:22]: I don't know about mailgun, but that comment fits spot on for Sendgrid, so either move or copy to Sendgrid.

File: tools_and_services/preferred_services.md [hunk 3]
```diff
@@ -1,15 +1,36 @@
...
 
-## Mailing
-[SendGrid](https://sendgrid.com/), in Heroku: use the free Starter addon (Automatically added by [Muffi](https://github.com/abtion/muffi)).
+Finding a balance between trying out a new service or using a well-known tool is tricky. There are several things to consider: can a tool that we previously used solve the problem? Is the learning curve of the new tool justified by the gained efficiency? Is the new (or old tool) GDPR compliant?
+As the matter is tricky, simply listing previously used tools is not enough. Instead, we have a three-color system.
+
+
+- ![](https://via.placeholder.com/10/00c900/?text=+) Green list: These services have been widely used at Abtion, and they usually provide the necessary functionality that we expect from them.
+If there is a service in this category, but the team wants to use a different one, the team must first check with the Principal Engineer or CTO.
+- ![](https://via.placeholder.com/10/ffff00/?text=+) Yellow list: We've used the services before, and we are satisfied. The services in this list should be the first option to consider on a team, although the team can decide to use a different tool if there is a good reason behind it.
```
Comment (runephilosof-abtion) [inline tools_and_services/preferred_services.md:9]: Shouldn't it rather be the services in the green list that are the first options to consider?

**Comment by hcarreras** [comment]
@KlausEnevoldsen-Abtion ideally, we add along the way. But to be fair, we need a strategy to update the guidelines. 
Suggestions are welcome (specially from the tech leadership group)

**Pull request merged**